src/panoptes/utils/utils.py

```diff
@@ -167,26 +167,30 @@ def string_to_params(opts):
     return args, kwargs
 
 
-def altaz_to_radec(alt=35, az=90, location=None, obstime=None, **kwargs):
+def altaz_to_radec(alt=None, az=None, location=None, obstime=None, **kwargs):
```

Comment:
Why have these as optional args with `None` as default? Would it be better to keep them as required positional arguments without default values?

Reply:
Mostly for backwards compatibility, although that's probably not required. It is a bit easier to pass things in as a dictionary if it accepts keyword arguments. 

Mostly I am just biased toward keyword args.